src/main/frontend/handler/ui.cljs

```diff
@@ -147,6 +147,17 @@
       (on-chosen (nth matched @current-idx) false)
       (and on-enter (on-enter state)))))
 
+(defn auto-complete-shift-complete
+  [state e]
+  (let [[matched {:keys [on-chosen on-shift-chosen on-enter]}] (:rum/args state)
+        current-idx (get state :frontend.ui/current-idx)]
+    (util/stop e)
+    (if (and (seq matched)
+             (> (count matched)
+                @current-idx))
+      (on-shift-chosen (nth matched @current-idx) false)
```

Comment:
Block references (when searching) don' have a `on-shift-chosen`, which can result in this error:
![CleanShot 2021-07-06 at 22 28 07](https://user-images.githubusercontent.com/479169/124617431-81981880-dea9-11eb-8017-327c5ff8f9fe.png)

Should us unify the same user experience for both page references and blocks references?

Reply:
Block ref shift+autocomplete now fallback to `on-chosen`